W-13729475 setup desktop duke (#78)
mulesoft/docs-code-builder: modules/ROOT/pages/implement-local-api-specification.adoc

Code Builder creates a workspace with both the API specification and Mule project and keeps them synchronized.

> Optionally, you can open the Command Palette by pressing Ctrl+Shift+p (Windows) or Cmd+Shift+p (Mac), and provide the following command:

Author: ```suggestion
Alternatively, open the Command Palette by pressing Ctrl+Shift+p (Windows) or Cmd+Shift+p (Mac), and provide the following command:
```
